windows-apps-src/debug-test-perf/deploying-and-debugging-uwp-apps.md

```diff
@@ -172,10 +175,11 @@ When you first register the layout from the network, your credentials will be ca
 You cannot select **keep all files on device** when you register the layout from the network because no files are physically copied to the remote device.
 
 > [!NOTE]
-> **Register layout from network** is currently supported on Xbox running Windows 10 Anniversary Update.
+> **Register layout from network** is currently supported on Xbox running Windows 10 Anniversary Update and PCs running Windows 10 Creators Update.
 
 On the remote device, the layout gets registered to the following default location depending on the device family:
-  `Xbox: \\MY-DEVKIT\DevelopmentFiles\XrfsFiles`
+  `Xbox: \\MY-DEVKIT\DevelopmentFiles\XrfsFiles` - this is a symlink to the **package registration path**
```

Comment:
Fine as is, it's a symlink, no need to hide that. Clarifies why it looks like files are being copied to disk.